src/framework/theme/components/context-menu/_context-menu.component.theme.scss

```diff
@@ -6,6 +6,15 @@
 @import '../../styles/core/mixins';
 
 @mixin nb-context-menu-theme() {
+
+  .context-menu-host {
```

Comment:
why don't we just put these styles to `.component.scss` to `:host {}` ?

Reply:
It's directive

Reply:
ops